packages/domains-table/src/utils/is-recently-registered.ts

```diff
@@ -1,8 +1,8 @@
-import moment from 'moment';
+import { isAfter, subMinutes } from 'date-fns';
 
 export function isRecentlyRegistered( registrationDate: string, numberOfMinutes = 30 ) {
 	return (
 		registrationDate &&
-		moment.utc( registrationDate ).isAfter( moment.utc().subtract( numberOfMinutes, 'minutes' ) )
+		isAfter( new Date( registrationDate ), subMinutes( new Date(), numberOfMinutes ) )
```

Comment:
I wouldn't have expected us to reach into the `domains-table` package for a utility like this. Seems like the sort of thing we'd have in the new `client/dashboard/utils/domain.ts` file.

On the other hand I do like seeing `domains-table` loosing one of its dependencies on `moment` 😃

Reply:
The domain utils currently exist in both the `domains-table` package and in `client/lib/domains/utils`. I’ve been trying to avoid introducing a new copy if possible.

Some of the other utils require a full domain object (not just raw endpoint data), so I moved those into `client/dashboard/utils/domain.ts`. But in this case, the utility is fairly simple, and I’d prefer to import it directly from the package if that’s an option.

I could also just duplicate it in `client/dashboard/utils/domain.ts`, but I’m not sure if that would provide any benefit.

Any thoughts?

Reply:
Can we move this function to `client/dashboard/utils/domain.ts` and then have existing references refer to it instead 😄

Reply:
Let's continue duplicating it for now, and revisit the best practice once all domain functionalities are implemented 🙂